docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
         environment:
             POSTGRES_PASSWORD: ${POSTGRESQL_PW}
             POSTGRES_USER: concordia
-            POSTGRES_MULTIPLE_DATABASES: concordia
+            POSTGRES_MULTIPLE_DATABASES: test_concordia
```

Comment:
I think this would have to be `concordia,test_concordia` in order for the various settings options to work with docker? e.g. the `concordia.settings_dev` and `concordia.settings_prod` mode (which is used in docker) have the db name as `concordia`

Nevermind - I tested and it works fine

Reply:
That's what I thought, too, but if you list the primary database in there the create command will crash due to it already existing